content/middlewares.md

```diff
@@ -244,3 +244,17 @@ await app.listen(3000);
 ```
 
 > info **Hint** Accessing the DI container in a global middleware is not possible. You can use a [functional middleware](middleware#functional-middleware) instead when using `app.use()`. Alternatively, you can use a class middleware and consume it with `.forRoutes('*')` within the `AppModule` (or any other module).
+
+#### Default body parser middleware
+
+When using the `express` adapter, the NestJS app will register `json` and `urlencoded` from the package `body-parser` by default. 
+- `urlencoded` is used with the `extended: true` flag and the optional `rawBody` flag if set in your `NestApplicationOptions`.
+- `body-parser` is used with the optional `rawBody` flag if set in your `NestApplicationOptions`.
+
+This can be turned off by setting the `bodyParser` flag to `false` when creating the application.
+
+```typescript
+const app = await NestFactory.create(AppModule, {bodyParser: false});
+```
+
+Without doing that, you won't be able to apply custom bodyParser middleware to individual routes.
```

Comment:
This isn't correct. AFAIR if you register a body-parser middleware by yourself - before initializing the application - the default one won't be registered, even without setting the `bodyParser` prop to false.